hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/SimpleCopyListing.java

```diff
@@ -18,30 +18,31 @@
 
 package org.apache.hadoop.tools;
 
-import org.apache.hadoop.fs.RemoteIterator;
-import org.apache.hadoop.thirdparty.com.google.common.collect.Lists;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.RemoteIterator;
+import org.apache.hadoop.fs.statistics.IOStatisticsLogging;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hdfs.protocol.SnapshotDiffReport;
 import org.apache.hadoop.io.SequenceFile;
 import org.apache.hadoop.io.IOUtils;
 import org.apache.hadoop.io.Text;
+import org.apache.hadoop.thirdparty.com.google.common.annotations.VisibleForTesting;
```

Comment:
now, these imports we are trying to leave up where they were. Because when cherrypicking we're trying to stay on the older versions. it's a PITA, I know